types/koji-tools/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for koji-tools 0.5
+// Project: https://github.com/madewithkoji/koji-tools
+// Definitions by: Jeff Peterson <https://github.com/bdjeffyp>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+// tslint:disable-next-line:export-just-namespace
+export = Koji;
+export as namespace Koji;
+
+declare namespace Koji {
+    // index.js methods
+    /**
+     * Sets up Koji.config parameters for each client and handles communication
+     * between the Koji live preview iframe and your app.
+     * @param options ???
+     */
+    function pageLoad(options?: {}): void;
```

Comment:
Missing few properties eg. from [usage](https://github.com/madewithkoji/koji-tools#usage---frontend) `Koji.config.colors.backgroundColor` and `Koji.config.strings.content` is suppose to exist?

Reply:
The `Koji.config` object is created from various JSON files into one JSON string, as specified in `./tools/buildConfig.js`. This makes any subelements of `Koji.config` and their types unknown. In the example you cited, the config object was built with two JSON files, named `colors.json` and `strings.json`. `buildConfig.js` bundles those JSON objects into the `Koji.config` object.

Reply:
This is why the `getConfig()` method returns an `any` type.

Reply:
I have changed the definition file to use `config` and specified it as an `object` type.